lib/Listerine/channels.ex

```diff
@@ -166,14 +166,16 @@ defmodule Listerine.Channels do
     |> prepend.(remove_course_channels(others))
   end
   
-  def role_list(_message, [], _mode) do
+  def manage_roles(_message, [], _mode) do
   end 
 
-  def role_list(message, [name | tail], mode) do
+  def manage_roles(message, [name | tail], mode) do
```

Comment:
This function should return the list of added/removed roles so you can then report to the user what roles were successfully added/removed. Similar to what was done [here](https://github.com/AmazingDevTeam/Listerine/blob/c0f56077ebe5a9beee7f4901904da92f1de6aba6/lib/Listerine/commands.ex#L27).